.travis.yml

```diff
@@ -4,11 +4,6 @@ services:
 - docker
 
 before_install:
-  - echo "deb http://us-central1.gce.archive.ubuntu.com/ubuntu/ trusty multiverse" | sudo tee /etc/apt/sources.list.d/print.list
-  - sudo apt-get update -yq
-  - sudo sh -c "echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections"
-  - sudo apt-get install -y ttf-mscorefonts-installer -qq
-  - fc-list | grep -i "Arial"
```

Comment:
Could you keep this check with the new font? Or did you confirm that the font is available on Travis?